.github/workflows/continuous-integration.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         run: composer install --prefer-dist --no-suggest --no-progress
 
       - name: Detect coding standard violations (PHPCS)
-        run: vendor/bin/phpcs -q --report=checkstyle --runtime-set ignore_errors_on_exit 1 --runtime-set ignore_warnings_on_exit 1 | cs2pr --graceful-warnings
+        run: vendor/bin/phpcs -q --report=checkstyle --runtime-set ignore_errors_on_exit 1 --runtime-set ignore_warnings_on_exit 1 --warning-severity=5 | cs2pr --graceful-warnings
```

Comment:
Hmm, isn't `5` the default?

Reply:
Indeed: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#filtering-errors-and-warnings-based-on-severity

> By default, PHP_CodeSniffer assigns a severity of 5 to all errors and warnings.

I somehow missed the warnings part there.